pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -58,7 +61,8 @@ let
     nativeBuildInputs = [ swig which ];
 
     buildInputs = [ python jemalloc openmpi glibcLocales numpy ]
-      ++ lib.optionals cudaSupport [ cudatoolkit cudnn nvidia_x11 ];
+      ++ lib.optionals cudaSupport [ cudatoolkit cudnn nvidia_x11 ]
```

Comment:
Mark `stdenv.isDarwin && cudaSupport` as broken.

Reply:
👍

Reply:
Would it be a good idea to remove the asserts at the top? (see https://github.com/NixOS/nixpkgs/issues/36229)